doozerlib/cli/job_pr_build_info.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import click
+import yaml
+from doozerlib.cli import cli
+from doozerlib.comment_on_pr import CommentOnPr
+
+
+@cli.command("comment-on-pr:from-job",
```

Comment:
I doubt this should be something in Doozer as a build tool. What is the use case here? Can this be a function of art-bot or a hack script in aos-cd-jobs?

Reply:
Hmm, the reason I kept in doozer is that when we add the feature for nightlies/releases, its gonna have some common code with art-bot, so I thought doozer could be the best place to keep this code so as to avoid redundancy.